core/auditlog/auditlog.go

```diff
@@ -9,13 +9,16 @@ import (
 	"io"
 	"math/rand"
 	"os"
+	"os/user"
 	"path/filepath"
+	"strconv"
 	"time"
 
+	"github.com/juju/juju/service/systemd"
```

Comment:
Hmm... this is a problem.

As a general rule, we don't import anything outside of a core package (but in juju/juju) from within a core package.

I think perhaps it makes sense to move the `SyslogUserGroup` function elsewhere. Probably a core package. It isn't systemd specific.

Looking through the core packages, there doesn't appear to be a good place to put it. But then I remembered `juju/paths` which deals with directories across systems. Storing file users and groups in a package that deals with directories is much closer.

How do you feel about moving the `juju/paths` package to `core/paths`?  It is imported in about 30 files. A key benefit of just moving the package is that nothing apart from the input line needs to change.